packages/react-devtools-shared/src/devtools/ProfilerStore.js

```diff
@@ -289,6 +289,10 @@ export default class ProfilerStore extends EventEmitter<{
   };
 
   onProfilingStatus: (isProfiling: boolean) => void = isProfiling => {
+    if (this._isProfiling === isProfiling) {
+      return;
+    }
+
```

Comment:
There are existing crashes reported on web:

https://github.com/facebook/react/issues/30661
https://github.com/facebook/react/issues/28838

Adding support for Fusebox in the PR encountered similar crashes.

These lines fix it for Fusebox; it could be the fix for web as well.